mass2/core/channels.py

```diff
@@ -359,15 +395,20 @@ def save_recipes(
         return steps
 
     def load_recipes(self, filename: str) -> "Channels":
+        """Return a copy of this Channels object with Recipe objects loaded from the given pickle file."""
         steps = mass2.misc.unpickle_object(filename)
         return self.with_steps_dict(steps)
 
     def parent_folder_path(self) -> pathlib.Path:
+        """Return the parent folder of the LJH files used to create this Channels object.
+        If there's more than one directory, the self.ch0 channel's directory is used."""
```

Comment:
There is no condition, `self.ch0` is always used.

Reply:
Okay, sure, but there's only a question of which, if there's more than one directory. But I'll reword.